src/Symfony/Component/Security/Core/Encoder/PlaintextPasswordEncoder.php

```diff
@@ -47,7 +47,7 @@ public function encodePassword($raw, $salt)
      */
     public function isPasswordValid($encoded, $raw, $salt)
     {
-        if ($this->isPasswordTooLong($raw)) {
+        if (null === $encoded || $this->isPasswordTooLong($raw)) {
```

Comment:
This change looks wrong to me. The interface declares that `$encoded` must be a `string`, but not `null`. Instead, we need to check where `isPasswordValid()` is called and prevent that call when there is no hash.

Reply:
it's already fixed in the new release 4.4.2. 
the test of $user::password is added in Symfony\Component\Security\Core\Authentication\Provider\DaoAuthenticationProvider 

https://github.com/symfony/symfony/blame/4e5b15353d643316fe9d47a0e2b654e7d706abb8/src/Symfony/Component/Security/Core/Authentication/Provider/DaoAuthenticationProvider.php#L58

![image](https://user-images.githubusercontent.com/33282931/71241304-93d45500-230b-11ea-803b-8b6e2e6db5f2.png)